### PR TITLE
v1.0

### PR DESCRIPTION
**Pull request by palkan**
- [x] Stop copy-pasting Rails internals.
- [x] Be a wrapper/extender, not patcher.
- [x] Use canonical Attributes API to define store attributes and their defaults.
- [x] **Drop Rails 5 and Ruby 2.5**

### CODE REVIEW

File: spec/cases/store_attribute_spec.rb [hunk 1]
```diff
@@ -135,11 +120,13 @@
       expect(jamie).to be_active
       expect(jamie.salary).to eq 12
 
+      jamie.salary = 13
```
Comment (palkan) [inline spec/cases/store_attribute_spec.rb:123]: Without updating store contents, Active Record doesn't include this attribute into a changeset (thus, it's not updated).

**Comment by palkan** [comment]
@markedmondson Please, take a look (or give this branch a try 🙂).

**Comment by markedmondson** [comment]
Oh whew that was a big change. I'll give it a test where we were seeing the issue, we don't have great testing around the other edge cases so hopefully nothing new has crept in, a run against our suite couldn't hurt though...

**Comment by palkan** [comment]
Thanks, @markedmondson!

I will take a look at this after the holidays 🎄

**Comment by palkan** [comment]
@markedmondson OK, I found the problem: Rails 6.0 :) It turned out, the internals have changed between 6.0 and 6.1.
Now 6.0 is supported.
